cmd/gin-repod/routes.go

```diff
@@ -29,4 +29,5 @@ func (s *Server) SetupRoutes() {
 	r.HandleFunc("/users/{user}/repos/{repo}/objects/{object}", s.getObject).Methods("GET")
 	r.HandleFunc("/users/{user}/repos/{repo}/browse/{branch}", s.browseRepo).Methods("GET")
 	r.HandleFunc("/users/{user}/repos/{repo}/browse/{branch}/{path:.*}", s.browseRepo).Methods("GET")
+	r.HandleFunc("/users/{user}/repos/{repo}/commits/{branch}", s.listRepoCommits).Methods("GET")
```

Comment:
NB: refs (i.e. branches) can contain slashes, i.e. "wip/foo/bar" is a valid branch name. I guess the two urls above are already broken too (this started out as a PoC after all).